app/helpers/tags_helper.rb

```diff
@@ -23,4 +31,10 @@ def html_class
   def body_id
     @body_id
   end
+
+  private
+
+  def article_locale
+    @article_locale ||= Locale.find_by(abbreviation: @article&.locale)
```

Comment:
Will this memoization persist past this request/article? I feel like we had locale / cache issues before.

Reply:
no, should only persist across page renders (the issue before was due to modifying `I18n.locale` directly)